lib/Doctrine/Migrations/Tools/TransactionHelper.php

```diff
@@ -15,6 +16,18 @@ final class TransactionHelper
     public static function commitIfInTransaction(Connection $connection): void
     {
         if (! self::inTransaction($connection)) {
+            Deprecation::trigger(
```

Comment:
it would be nice to know which migration triggered the error... but i guess that here there is not enough context to know that... :(